frontend/util.lua

```diff
@@ -330,6 +330,24 @@ function util.tableSize(t)
     return count
 end
 
+--- Like pairs, but with sorted keys.
+---- @param t Lua table
+---- @param fn Lua function
+---- @treturn Lua function iterator on table t
+function util.opairs(t, fn)
+    local _t = {}
+    for k in pairs(t) do
+        _t[#_t + 1] = k
+    end
+    table.sort(_t, fn)
+    local i = 0
+    return function()
+        i = i + 1
+        local k = _t[i]
+        if k then return k, t[k] end
+    end
+end
+
```

Comment:
We already have [ffiUtil.orderedPairs](https://github.com/koreader/koreader-base/blob/ec3d5021852a1acf3471364a020d07b7de439356/ffi/util.lua#L675-L678) ;).